test/parallel/test-net-server-listen-options.js

```diff
@@ -42,6 +42,7 @@ const listenOnPort = [
     listen('0', common.mustCall(close));
     listen(0, common.mustCall(close));
     listen(undefined, common.mustCall(close));
+    listen(null, common.mustCall(close));
```

Comment:
It may not be obvious, but because of the loop, this tests both cases that were removed below.